team-manager/src/main/java/ua/knu/knudev/teammanager/requirements/PreMasterDeveloperRequirements.java

```diff
@@ -0,0 +1,12 @@
+package ua.knu.knudev.teammanager.requirements;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class PreMasterDeveloperRequirements {
```

Comment:
Make as record please

Reply:
can`t because i need setters here, we already discussed all this situations with records